### PR TITLE
NMS-14632: Add alt text to Operation (pt. 1)

### PR DESCRIPTION
**Pull request by emarsh1995**
- Update `operation/deep-dive/provisioning/auto-discovery.adoc`, `/directed-discovery.adoc`, `/foreign-source.adoc`
- Update `operation/deep-dive/service-assurance/application-perspective-monitoring/introduction.adoc`
- Update `operation/deep-dive/service-assurance/critical-service.adoc`, `/downtime-model.adoc`, `/introduction.adoc`, `/path-outages.adoc`, `/polling-packages.adoc`, `/configuration.adoc`
- Update `operation/deep-dive/thresholds/datasource.adoc`, `/thresholding.adoc`
- Update `operation/deep-dive/user-management/pre-authentication.adoc`, `/security-roles.adoc`, `/user-oncall.adoc`
- Update `operation/quick-start/inventory.adoc`, `/notifications.adoc`, `/service-assurance.adoc`, `/thresholding.adoc`

### All Contributors

* [x] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [x] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### Contribution Checklist

* Please [make an issue in the OpenNMS issue tracker](https://issues.opennms.org) if there isn't one already.<br />Once there is an issue, please:
  1. update the title of this PR to be in the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
  4. once you've created this PR, please link to it in a comment in the JIRA issue
  Don't worry if this sounds like a lot, we can help you get things set up properly.
* **If this code is likely to affect the UI, did you name your branch with `-smoke` in it to trigger smoke tests?**
* If this is a new or updated feature, is there documentation for the new behavior?
* If this is new code, are there unit and/or integration tests?
* If this PR targets a `foundation-*` branch, does it try to avoid changing files in `$OPENNMS_HOME/etc/`?

### What's Next?

A PR should be assigned at least 2 reviewers.  If you know that someone would be a good person to review your code, feel free to add them.

If you need help making additions or changes to the documentation related to your changes, please let us know.

In any case, if anything is unclear or you want help getting your PR ready for merge, please don't hesitate to say something in the comments here, or in [the #opennms-development chat channel](https://chat.opennms.com/opennms/channels/opennms-development).

Once reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the OGP) you are welcome to merge the PR when you're ready. Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): https://opennms.atlassian.net/browse/NMS-14632

### CODE REVIEW

File: docs/modules/operation/pages/deep-dive/service-assurance/polling-packages.adoc [hunk 1]
```diff
@@ -163,7 +154,7 @@ In the previous example, all IP interfaces in `192.168.1/24` or `2600:/64` will
 The IP interface and service pages show which polling package and service configuration are applied for this specific service.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/deep-dive/service-assurance/polling-packages.adoc:154]:
```suggestion
The IP interface and service pages show which polling package and service configuration are applied for this specific service, and associated statistics.
```
Add this only if you agree with comment below.

File: docs/modules/operation/pages/deep-dive/service-assurance/polling-packages.adoc [hunk 2]
```diff
@@ -180,80 +171,32 @@ If a poller is matched using its pattern, the parts of the service name that mat
...
-                Poller package
-        -S, --service
-                Service name
-        -n, --node-id
-                Node ID for service
-        -c, --class
-                Monitor class
 
-----
+NOTE: In the shell, you can view the command's help information by using `opennms:poll --help`.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/deep-dive/service-assurance/polling-packages.adoc:190]:
```suggestion
NOTE: In the shell, you can use `opennms:poll --help` to view the command's help information.
```

File: docs/modules/operation/pages/deep-dive/service-assurance/polling-packages.adoc [hunk 3]
```diff
@@ -180,80 +171,32 @@ If a poller is matched using its pattern, the parts of the service name that mat
...
 
-.Run ICMP monitor configuration defined in specific polling package
 [source, console]
-----
 opennms> opennms:poll -S ICMP -P example1 10.23.42.1
-----
 
 The output is verbose, which lets you debug monitor configurations.
-Important output lines:
+Important output lines are as follows:
```
Comment (Bonrob2) [inline docs/modules/operation/pages/deep-dive/service-assurance/polling-packages.adoc:198]:
```suggestion
Important output lines include the following:
```

File: docs/modules/operation/pages/deep-dive/service-assurance/polling-packages.adoc [hunk 4]
```diff
@@ -276,40 +220,24 @@ Service is Up on 192.168.31.100 using org.opennms.netmgt.poller.monitors.IcmpMon
...
+For more information, see xref:reference:configuration/filters/filters.adoc[].
 
+.SSH to localhost
 [source, console]
-----
 ssh -p 8101 admin@localhost
-----
 
-Once in the shell, show command help as follows:
+NOTE: In the shell, you can view the command's help information by using `opennms:filter --help`.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/deep-dive/service-assurance/polling-packages.adoc:233]:
```suggestion
NOTE: In the shell, you can use `opennms:filter --help` to view the command's help information.
```

File: docs/modules/operation/pages/deep-dive/thresholds/datasource.adoc [hunk 5]
```diff
@@ -74,7 +77,8 @@ To determine the name of the data source, navigate to the *Resource Graphs* scre
 . Scroll through the graphs to find the title of the graph that displays the metric on which you want to threshold.
 For example, "Number of Processes" or "System Uptime":
 +
-image::thresholding/Graphs.png[]
+.{page-component-title} performance data metric graphs
+image::thresholding/Graphs.png["Two example performance data metric graphs: one displaying system uptime, and one displaying the number of active processes over time"]
```
Comment (Bonrob2) [inline docs/modules/operation/pages/deep-dive/thresholds/datasource.adoc:81]:
```suggestion
image::thresholding/Graphs.png["Two example performance data metric graphs: one displays system uptime, and one displays the number of active processes over time"]
```

File: docs/modules/operation/pages/deep-dive/thresholds/thresholding.adoc [hunk 6]
```diff
@@ -1,29 +1,29 @@
...
 
-image::thresholding/thresholding-flow.png[]
+Collectd gathers and stores performance data from collectors.
+This is the data against which {page-component-title} evaluates thresholds.
+Thresholds trigger events when specific values are met, and notifications and alarms can be configured for these events.
 
-{page-component-title} uses four thresholding algorithms to trigger an event.
-See xref:quick-start/thresholding.adoc[Set Up a Threshold] in the Quick Start section for more information.
+.Thresholding network diagram
+image::thresholding/thresholding-flow.png["Network diagram showing the relationships between thresholding components in {page-component-title}"]
```
Comment (Bonrob2) [inline docs/modules/operation/pages/deep-dive/thresholds/thresholding.adoc:23]:
```suggestion
image::thresholding/thresholding-flow.png["Network diagram showing the relationships among thresholding components in {page-component-title}"]
```

File: docs/modules/operation/pages/deep-dive/user-management/pre-authentication.adoc [hunk 7]
```diff
@@ -1,63 +1,58 @@
...
-
-Define the pre-authentication configuration in `$\{OPENNMS_HOME}/jetty-webapps/opennms/WEB-INF/spring-security.d/header-preauth.xml`.
+You can configure your {page-component-title} instance to run behind a proxy that provides authentication.
+Authenticated users can then be directed to the {page-component-title} web UI using a header.
+You can configure pre-authentication settings in `$\{OPENNMS_HOME}/jetty-webapps/opennms/WEB-INF/spring-security.d/header-preauth.xml`.
 This file is automatically included in the Spring security context, but is not enabled by default.
 
-WARNING: Do not configure {page-component-title} this way unless you are certain the web UI is accessible only to the proxy and not to end users.
-Otherwise, malicious attackers can craft queries that include the pre-authentication header and get full control of the web UI and REST APIs.
+WARNING: Do not configure {page-component-title} to use pre-authentication unless you are certain that the web UI is accessibly only to the proxy, and not to end users.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/deep-dive/user-management/pre-authentication.adoc:9]:
```suggestion
WARNING: Do not configure {page-component-title} to use pre-authentication unless you are certain that the web UI is accessible only to the proxy, and not to end users.
```

File: docs/modules/operation/pages/deep-dive/user-management/pre-authentication.adoc [hunk 8]
```diff
@@ -1,63 +1,58 @@
...
 | enabled
-| Whether the pre-authentication plugin is active.
+| Indicates whether the pre-authentication plugin is active.
 | false
 
 | failOnError
-| If `true`, disallow login if the header is not set or the user does not exist.
-If `false`, fall through to other mechanisms (basic auth, form login, and so on).
+| If `true`, block users from logging in if their header is not set or the user does not exist. +
+If `fales`, fall through to other mechanisms (basic authentication, form login, and so on).
```
Comment (Bonrob2) [inline docs/modules/operation/pages/deep-dive/user-management/pre-authentication.adoc:33]:
```suggestion
If `false`, fall through to other mechanisms (basic authentication, form login, and so on).
```

File: docs/modules/operation/pages/deep-dive/user-management/pre-authentication.adoc [hunk 9]
```diff
@@ -1,63 +1,58 @@
...
 | false
 
 | userHeader
-| The HTTP header that specifies the user to authenticate as.
+| The HTTP header that specifies which user is authenticated.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/deep-dive/user-management/pre-authentication.adoc:37]:
```suggestion
| The HTTP header that specifies which user to authenticate.
```

File: docs/modules/operation/pages/deep-dive/user-management/pre-authentication.adoc [hunk 10]
```diff
@@ -66,11 +61,7 @@ Modify `$\{OPENNMS-HOME}/jetty-webapps/opennms/WEB-INF/applicationContext-spring
 </beans:bean>
 ----
 
-You can then use the following command to test:
+After the pre-authentication plugin is enabled and set up, you can test its configuration using the following console command:
```
Comment (Bonrob2) [inline docs/modules/operation/pages/deep-dive/user-management/pre-authentication.adoc:64]:
```suggestion
After the pre-authentication plugin is enabled and set up, use the following command to test its configuration:
```

File: docs/modules/operation/pages/deep-dive/service-assurance/application-perspective-monitoring/introduction.adoc [hunk 11]
```diff
@@ -1,47 +1,42 @@
...
+Application Perspective Monitoring uses the Minion infrastructure to monitor a service's availability from different perspectives.
+When a service is not responsive, an outage is generated with the corresponding perspective information.
 
-.Application Perspective Monitoring operation
-image::application-perspective-monitoring/apm-operation.png[]
+.Application Perspective Monitoring diagram
+image::application-perspective-monitoring/apm-operation.png["Network diagram illustrating how remote services are monitored with Application Perspective Monitoring", 700]
 
-The service monitor configuration is looked up in the `poller-configuration.xml`.
+The service monitor configuration is looked up in `poller-configuration.xml`.
```
[the remark this answers is not in the document]
Comment (emarsh1995) [reply docs/modules/operation/pages/deep-dive/service-assurance/application-perspective-monitoring/introduction.adoc:14]: @mmahacek I'm assuming that, in this case, the Minion is looking up the service monitor configuration. Is this the case?

[the remark this answers is not in the document]
Comment (mmahacek) [reply docs/modules/operation/pages/deep-dive/service-assurance/application-perspective-monitoring/introduction.adoc:14]: Minions don't see the service monitor configs.  The core server (Horizon/Meridian) reads the config and puts a job request message on the broker (ActiveMQ/Kafka).  That request has everything a Minion would need to do that task.
All that said, I'm going to suggest removing this sentence, as it's not really necessary to state this fact, and it's kind of dangling below the image by itself.

File: docs/modules/operation/pages/deep-dive/service-assurance/application-perspective-monitoring/introduction.adoc [hunk 12]
```diff
@@ -1,47 +1,42 @@
...
```
Comment (emarsh1995) [inline docs/modules/operation/pages/deep-dive/service-assurance/application-perspective-monitoring/introduction.adoc:15]:
```suggestion
```

**Review by Bonrob2 — APPROVED** [review]
LGTM! Thanks.

**Pull request merged**